README.md

```diff
@@ -1,19 +1,21 @@
 # High Availability Postgres on Fly
 
-This is a ready to go HA Postgres app that runs on Fly.
+This is a ready-to-go HA Postgres app that runs on Fly.
 
 ## Prepare your app
 
 ### `flyctl init`
 Init gets you going with a Fly application and generates a config file.
 
 ### Set secrets
-This app requires `SU_PASSWORD` and `REPL_PASSWORD` environment variables.
+This app requires `SU_PASSWORD` and `REPL_PASSWORD` environment variables and gives users the option to create a "root" account on new database clusters using 'OPERATOR_PASSWORD'.
```

Comment:
Let's replace the single quotations with backtick's for consistency.  E.G. `OPERATOR_PASSWORD` instead of 'OPERATOR_PASSWORD'. 

> and gives users the option to create a "root" account on new database clusters

We could maybe make this a bit more direct.  

This app requires `SU_PASSWORD` and `REPL_PASSWORD` environment variables.  Optionally, you may specify the `OPERATOR_PASSWORD` to seed the initial "postgres" user on provision.